check-postgresql/lib/check-postgresql.go

```diff
@@ -30,6 +31,9 @@ func (p postgresqlSetting) getDriverAndDataSourceName() (string, string) {
 		dbName = p.Database
 	}
 	dataSourceName := fmt.Sprintf("user=%s password=%s host=%s port=%s dbname=%s sslmode=%s connect_timeout=%d", p.User, p.Password, p.Host, p.Port, dbName, p.SSLmode, p.Timeout)
+	if p.SSLRootCert != "" {
+		dataSourceName += fmt.Sprintf(" sslrootcert=%s", p.SSLRootCert)
```

Comment:
`sslrootcert=` parameter should be passed to `lib/pq` if and only if the `--sslrootcert` option has been set explicitly since that does not have the default value.